externs/shaka/net.js

```diff
@@ -125,7 +125,9 @@ shakaExtern.Response;
  *
  * @typedef {!function(string,
  *                     shakaExtern.Request,
- *                     shaka.net.NetworkingEngine.RequestType):
+ *                     shaka.net.NetworkingEngine.RequestType,
+ *                     function(ProgressEvent,
```

Comment:
Outputting `ProgressEvent`s might be a little tricky if we start using `Fetch`, but it looks like we may be able to synthesize these events if we do.

Just thinking out loud.